index.html

```diff
@@ -1131,6 +1131,17 @@
             the <a>media element state</a> and <a>remote playback state</a> in
             sync (unless <a>media mirroring</a> is used).
           </div>
+          <div class="note">
+            The user agent SHOULD pause local audio and video output of the
+            media element while remote playback
+            is <a data-link-for="RemotePlaybackState">connected</a>.
+          </div>
+          <div class="note">
+            If the user agent is <a>exposing a user interface to the user</a>
+            for the media element (i.e., using default controls), the user agent
+            SHOULD convey the fact that remote playback is connected through an
```

Comment:
nit: I don't think "remote playback is connected" is a valid expression, at least in the spec. Maybe replace with "remote playback state is connected" with links for both state and connected?